### PR TITLE
v1.1.0 | Daskeyboard Applet Crypto Watch

### PR DESCRIPTION
**Pull request by Wolfe1**
Release notes: https://github.com/Wolfe1/daskeyboard-applet--crypto-watch/releases/tag/v1.1.0

### CODE REVIEW

**Comment by Wolfe1** [comment]
@daskeyboard

**Review by fabiendv — CHANGES_REQUESTED** [review]
Great Job! I love it.

Just a few things to do:

- [x] Remove 2 `console.log` 
- [x] Add package-lock.json into your .gitignore
- [x] Run `yarn upgrade`. It will fix your issue :link: https://github.com/Wolfe1/daskeyboard-applet--crypto-watch/issues/229

After this quick update, I'll merge right away :rocket:

**Comment by fabiendv** [comment]
I went ahead and completed changes.
Please update your repo :)

**Pull request merged**
